src/components/ProgressBar/__tests__/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+
+import { NAMESPACE } from '../../../constants'
+import { ProgressBar } from '../'
+
+describe('<ProgressBar />', () => {
+  it('renders a default className', () => {
+    const $ = shallow(<ProgressBar>_</ProgressBar>)
+    expect($.hasClass(`${NAMESPACE}c-progress-bar`)).toBe(true)
+  })
+
+  it('passes in an optional className override', () => {
```

Comment:
This test should be combined with the first because how do we know that the default classname is not being overridden?
I would also write in the description: 'takes a className that gets appended` There's no override happening here, and the wording would be confusing should we come back to this at a later date.

Reply:
done